Bookmark-Manager/src/main/java/com/bookmarkmanager/bookmark/BookMarkController.java

```diff
@@ -0,0 +1,46 @@
+package com.bookmarkmanager.bookmark;
+
+
+import com.bookmarkmanager.pojo.Bookmark;
+import lombok.RequiredArgsConstructor;
+
+import java.net.URI;
+import java.util.UUID;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/bookmarks")
+@RequiredArgsConstructor
+public class BookMarkController {
+  private final BookmarkService bookmarkService;
+
+
+  // Create a new bookmark
+  @PostMapping
+  public ResponseEntity<UUID> createBookmark(@RequestBody Bookmark bookmark) {
+    Bookmark createdBookmark = bookmarkService.addBookmark(bookmark);
```

Comment:
We shouldn't use entity objects in controller layer

Reply:
Thank you for the feedback!. I have added BookmarkDTO.